dockers/docker-orchagent/tunnel_packet_handler.py

```diff
@@ -270,66 +274,68 @@ def sniffer_restart_required(self, messages):
                     return True
         return False
 
+    def start_sniffer(self):
+        """
+        Starts an AsyncSniffer and waits for it to inititalize fully
+        """
+        self.sniffer = AsyncSniffer(
+            iface=self.sniff_intfs,
+            filter=self.packet_filter,
+            prn=self.ping_inner_dst,
+            store=0
+        )
+        self.sniffer.start()
+
+        while not hasattr(self.sniffer, 'stop_cb'):
+            time.sleep(0.1)
```

Comment:
I think, we can have a 1second wait here as 0.1 may be too aggressive

Reply:
Did some testing, it looks like it takes about 0.2 seconds to initialize, might be ok to keep 0.1 seconds so we can start the service ASAP?

Reply:
ok